lizmap/modules/admin/locales/en_US/admin.UTF-8.properties

```diff
@@ -263,10 +263,11 @@ server.information.qgis.plugin.action=Action
 server.information.qgis.test.ok=QGIS Server is correctly installed and returns the expected response for OGC requests.
 server.information.qgis.test.error=QGIS Server is not correctly installed in your server or the URL for OGC requests given in Lizmap configuration is not correct.
 server.information.qgis.error.fetching.information=We cannot get the details about your QGIS Server installation (version, plugins, etc.).
-server.information.qgis.error.fetching.information.detail=Either the version of the installed QGIS Server is under %s, or \
-the Lizmap server plugin is not detected in your server, or is installed with a version under %s. \
-Please upgrade QGIS Server to minimum %s and install or upgrade the Lizmap server plugin to \
-minimum %s by reading the documentation about the environment variable
+server.information.qgis.error.fetching.information.description=You must check the followings steps are done :
+server.information.qgis.error.fetching.information.qgis.version=<strong>QGIS Server</strong> is above or equal to <strong>%s</strong>
+server.information.qgis.error.fetching.information.plugin.version=The <strong>Lizmap QGIS server plugin</strong> is above or equal to <strong>%s</strong>
```

Comment:
```suggestion
server.information.qgis.error.fetching.information.plugin.version.html=The <strong>Lizmap QGIS server plugin</strong> is above or equal to <strong>%s</strong>
```

Reply:
You need to add .html to the variable name in the properties files too

Reply:
Ok indeed !

So there is a difference, both these strings are working : 

```
server.information.qgis.error.fetching.information.plugin.version.html=The <strong>Lizmap QGIS server plugin</strong> is above or equal to <strong>%s</strong>
server.information.qgis.error.fetching.information.environment.variable=The <strong>environment variable</strong> described in the documentation link below is correctly set in your configuration
```

It depends if there is a variable to replace. OK ;-)

Reply:
i didn't know this detail , the big deal : jlocale doesn't behave as `{@XX@}` localisalisation in template :(